DashAI/front/src/components/ConfigurableObject/FormInputs.jsx

```diff
@@ -0,0 +1,377 @@
+import React, { useState, useEffect } from "react";
+import {
+  TextField,
+  MenuItem,
+  Stack,
+  Typography,
+  IconButton,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Button,
+  FormControlLabel,
+  Checkbox,
+  FormControl,
+} from "@mui/material";
+import SettingsIcon from "@mui/icons-material/Settings";
+import { generateTooltip } from "../ParameterForm";
+import PropTypes from "prop-types";
+import { getDefaultValues } from "../../utils/values";
+import Subform from "./Subform";
+import { styled } from "@mui/material/styles";
+
+const Input = styled(TextField)(({ theme }) => ({
+  minWidth: "20vw",
+}));
+
+export function Integer({ name, value, onChange, description, error }) {
```

Comment:
Move to `front/src/components/ConfigurableObject/Inputs/IntegerInput.jsx`

```suggestion
export function IntegerInput({ name, value, onChange, description, error }) {
```